raiden/transfer/channel.py

```diff
@@ -1745,11 +1755,16 @@ def handle_receive_lockedtransfer_light(
         channel_state.partner_state.balance_proof = mediated_transfer.balance_proof
         channel_state.partner_state.merkletree = merkletree
 
+        print("raiden/transfer/channel.py:1806 PARTNER STATE UPDATED WITH BP {}".format(channel_state.partner_state.balance_proof))
+
         lock = mediated_transfer.lock
         channel_state.partner_state.secrethashes_to_lockedlocks[lock.secrethash] = lock
-        events = []
-
-
+        send_processed = SendProcessed(
```

Comment:
can you explain the reason for this change? (empty list to non-empty list)

Reply:
I did that above in the other case